xaynetwork/xayn_discovery_app
# Analytics script to check data integrity between amplitude and mixpanel

### What 🕵️ 🔍
- Retrieve user data with DistinctId for Amplitude and Mixpanel. Works only for exported users.

----------

### How to test (please adjust template) 🥼 🔬
- Run the script `python3 retrieve_data_from_analytics.py --distinctId="INSERT_ID" --env="dev" --start="START_DATE" --end="END_DATE"`. Make sure that end date is the last date the user sent an event to amplitude before we migrated the data.

----------
